ts/features/wallet/component/__test__/CardLayoutPreview.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render } from "@testing-library/react-native";
+import { View } from "native-base";
+import * as React from "react";
+import { CardLayoutPreview } from "../CardLayoutPreview";
+
+describe("CardLayoutPreview", () => {
+  it("should show the left and the right received components", () => {
+    const left = () => <View testID="leftComponent"></View>;
```

Comment:
```suggestion
    const left = () => <View testID="leftComponent"/>;
```

Reply:
Modified in [6faa89f](https://github.com/pagopa/io-app/pull/2878/commits/6faa89f07670a693b243ff6c5da2bac028d52962)